src/libraries/icubmod/embObjMotionControl/embObjMotionControl.cpp

```diff
@@ -1043,11 +1043,18 @@ bool embObjMotionControl::fromConfig_Step2(yarp::os::Searchable &config)
             _temperatureSensorsVector.at(j) = std::make_unique<eomc::TemperatureSensorNONE>();
         }
     }
-    
 
+    int defaultTimeout = 100;
+
+    if (this->serviceConfig.ethservice.configuration.type == eomn_serv_MC_advfoc)
+    {
+        // temporary workaround
+        // in this case the default timeout is 300 ms because there is some lag in AMC
+        defaultTimeout = 300;
+    }
```

Comment:
Just pointing this out for those working mostly on AMC family as this required change of the timeout default value should trigger out attention at least.

cc @marcoaccame @mfussi66 

At any rate, this comment is not blocking for merging the PR.